eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToDbsnpVariantsWrapperProcessorTest.java

```diff
@@ -489,10 +499,10 @@ public void transformMicrosatelliteCornerCases() throws Exception {
                                         TAXONOMY);
 
         variants = processor.process(subSnpNoHgvs).getSubmittedVariants();
-        assertProcessedVariant(subSnpNoHgvs, variants.get(0), CHROMOSOME, CHROMOSOME_START, "ACACACACAC", "(AC)11",
-                               false, false, 1);
-        assertProcessedVariant(subSnpNoHgvs, variants.get(1), CHROMOSOME, CHROMOSOME_START, "ACACACACAC", "(AC)14",
-                               false, false, 1);
+        assertProcessedVariant(subSnpNoHgvs, variants.get(0), CHROMOSOME, CHROMOSOME_START, "ACACACACAC",
+                               "ACACACACACACACACACACAC", false, false, 1);
+        assertProcessedVariant(subSnpNoHgvs, variants.get(1), CHROMOSOME, CHROMOSOME_START, "ACACACACAC",
+                               "ACACACACACACACACACACACACACAC", false, false, 1);
```

Comment:
Based on the current regexp definition, (TG) is a valid input (matches regex "([a-zA-Z]+)\d*"). But an exception is thrown when a test is run against that input. Also, test fails against this input: https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs718886078

Reply:
Also, the regex seems to be permissive. For ex: "TG Jibb_1$_erish GTA" when unrolled becomes "TGJibberishGTA". Not sure if it is a bad thing but I was wondering if it would be worthwhile to throw errors in such cases?